app/assets/stylesheets/button-group-custom.scss

```diff
@@ -1,3 +1,5 @@
+@use "govuk-frontend/dist/govuk/all" as *; // This means we can use GOV.UK variables i.e. `$govuk-etc`
```

Comment:
Ah nice work on the conversation from imports to `@use`

If you're going to assign a variable, it should probably be more descriptive than an wild card, something like https://github.com/ministryofjustice/moj-flask-govuk-skeleton/blob/main/app/static/src/scss/styles.scss would be a better use of a variable name description.

I don't think you even need to do that though, you can probably do the below if you're not utlising scss variables. 

`@use "govuk-frontend/dist/govuk/all"`

Example here in a PR https://github.com/ministryofjustice/govuk-frontend-express/blob/feature/cloud-platform-deploy/src/scss/main.scss

Reply:
@psweeting1 I've tried both ways but I get `Error: Undefined variable.`, if I don't use a wildcard.